libkineto/CMakeLists.txt

```diff
@@ -133,13 +133,91 @@ IF (NOT ROCM_SOURCE_DIR AND NOT ROCTRACER_INCLUDE_DIR)
     set(LIBKINETO_NOROCTRACER ON CACHE BOOL "" FORCE)
 endif()
 
+# Set LIBKINETO_NOXPUPTI to explicitly disable XPUPTI
+# Otherwise, XPUPTI is disabled if not found
+if(NOT LIBKINETO_NOXPUPTI)
+    if (NOT SYCL_INCLUDE_DIR OR NOT SYCL_LIBRARY)
```

Comment:
why we do not follow IPEX as below?
`find_package(IntelSYCL REQUIRED)`

Reply:
ok

Reply:
It's a question, not a request.